specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-05-01/migrate.json

```diff
@@ -0,0 +1,3748 @@
+      "type": "object",
+      "description": "A REST API operation supported by the provider.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the operation.",
+          "readOnly": true
+        },
+        "display": {
+          "description": "Displayable properties of the operation.",
+          "$ref": "#/definitions/OperationDisplay"
+        },
+        "origin": {
+          "type": "string",
+          "description": "Origin of the operation.",
+          "readOnly": true
+        }
+      }
+    },
+    "DownloadUrl": {
+      "type": "object",
+      "description": "Download URL for assessment report.",
+      "properties": {
+        "assessmentReportUrl": {
+          "type": "string",
+          "description": "Hyperlink to download report.",
+          "readOnly": true
+        },
+        "expirationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Expiry date of download url.",
+          "readOnly": true
+        }
+      }
+    },
+    "ProjectResultList": {
+      "type": "object",
+      "description": "List of projects.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of projects.",
+          "items": {
+            "$ref": "#/definitions/Project"
+          }
+        }
+      }
+    },
+    "MachineResultList": {
+      "type": "object",
+      "description": "List of machines.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of machines.",
+          "items": {
+            "$ref": "#/definitions/Machine"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "GroupResultList": {
+      "type": "object",
+      "description": "List of groups.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of groups.",
+          "items": {
+            "$ref": "#/definitions/Group"
+          }
+        }
+      }
+    },
+    "AssessedMachineResultList": {
+      "type": "object",
+      "description": "List of assessed machines.",
+      "properties": {
+        "value": {
+          "description": "List of assessed machines.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssessedMachine"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "AssessmentResultList": {
+      "type": "object",
+      "description": "List of assessments.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of assessments.",
+          "items": {
+            "$ref": "#/definitions/Assessment"
+          }
+        }
+      }
+    },
+    "CollectorBodyAgentSpnProperties": {
+      "type": "object",
+      "properties": {
+        "authority": {
+          "type": "string"
```

Comment:
Could we add description for these propeties?

Reply:
Done.